tests/common.py

```
@@ -0,0 +1,49 @@
+    r = Redis(port=port)
+    try:
+        r.ping()
+        client = FalkorDB(port=port)
+        g = client.select_graph("test")
+        return
+    except:
+        shutdown = True
+        if os.path.exists("redis-test.log"):
+            os.remove("redis-test.log")
+        redis_server = subprocess.Popen(executable="/usr/local/bin/redis-server",
+                                        args=["--save", "", "--port", port, "--logfile", "redis-test.log",
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)